app/services/preserved_object_handler.rb

```diff
@@ -133,18 +133,28 @@ def update_version
 
   private
 
-  def create_db_objects(status)
+  def create_db_objects(status, validates=false)
     results = []
     pp_default = PreservationPolicy.default_preservation_policy
     create_results = with_active_record_rescue do
       po = PreservedObject.create!(druid: druid,
                                    current_version: incoming_version,
                                    preservation_policy: pp_default)
-      PreservedCopy.create!(preserved_object: po,
-                            version: incoming_version,
-                            size: incoming_size,
-                            endpoint: endpoint,
-                            status: status)
+      pc_attrs = {
+        preserved_object: po,
+        version: incoming_version,
+        size: incoming_size,
+        endpoint: endpoint,
+        status: status
+      }
+
+      if validates
+        t = Time.current
+        # Returns the value of time as an integer number of seconds since the Epoch.
+        pc_attrs[:last_audited] = t.to_i
```

Comment:
I'm guessing `last_audited` is meant to be the integer format for easy sorting and comparisons and so on.